EIPS/eip-.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Multi-chain Deterministic Deployment Factory
```

Comment:
```suggestion
title: Deterministic Deployment Factory precompile
```
Suggest not including multi-chain in an EIP title.
Suggest explicitly stating that it is a precompile. (I am not an EIP editor)

Reply:
I want to avoid calling this a precompile, at least for now, because it has very different characteristics.